anybox/recipe/odoo/utils.py

```diff
@@ -85,8 +85,8 @@ def major_version(version_string):
     if m is None:
         raise ValueError("Unparseable version string: %r" % version_string)
 
-    major = int(m.group(1))
-    minor = m.group(3)
+    major = int(m.group(2))
+    minor = m.group(4)
```

Comment:
And a test for the new cases [here](https://github.com/anybox/anybox.recipe.odoo/blob/master/tests_with_odoo/test_version.py#L46)